docs/reference/data-frames/apis/get-transform-stats.asciidoc

```diff
@@ -42,7 +44,11 @@ The API returns the following information:
 `transforms`::
   (array) An array of statistics objects for {dataframe-transforms}.
 
-//==== Authorization
+==== Authorization
+
+If the {es} {security-features} are enabled, you must have
+`data_frame_transforms_user` cluster privileges to use this API. For more
```

Comment:
The specific cluster privilege is `monitor_data_frame_transforms`. A role that has that privilege is `data_frame_transforms_user`